core/app/models/spree/return_authorization.rb

```diff
@@ -20,6 +21,8 @@ class ReturnAuthorization < Spree::Base
     validate :must_have_shipped_units, on: :create
     validate :no_previously_exchanged_inventory_units, on: :create
 
+    validates :recipient, presence: true, if: :override_recipient
+    attr_accessor :override_recipient
```

Comment:
We need the volatile attribute because of the way the interface works. See attached screenshot (Please ignore styling for now). We expose a checkbox allowing users to override the reimbursement recipient. This is handy in the case where a user checks the box but doesn't select a user.

![fullscreen_11_25_15__4_43_pm](https://cloud.githubusercontent.com/assets/24883/11410257/ef286862-9393-11e5-9b1d-9d12ea09644a.png)